pkgs/top-level/all-packages.nix

```diff
@@ -13352,6 +13352,7 @@ in
   gnutls = callPackage ../development/libraries/gnutls/default.nix {
     inherit (darwin.apple_sdk.frameworks) Security;
     util-linux = util-linuxMinimal; # break the cyclic dependency
+    autoconf = autoconf269;
```

Comment:
Please use buildPackages.autoconf269 to not break cross

Reply:
All the other pins in all-packages use `autoconf = autoconf269;`, is that going to cause problems? I see that autoreconfHook is made to use buildPackages but not so for autoconf itself

Reply:
See bf46afde6b22942ea605983f66ee23038986ba07, this should be done for all nativeBuildInputs that are called with an explicit argument.

Reply:
made the treewide change in 136c0d01d25b030c0d103e989de2d5340904dc51